src/Internal/Proxy/LegacyProxy.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Class LegacyProxy
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Proxy;
+
+use Exception; // ToDo: Replace this with another exception.
```

Comment:
May better use `ProxyException` now? I don't think we will visit this class often after this PR is merged.

Reply:
Good point, done in https://github.com/Automattic/woocommerce-payments/pull/6908/commits/0af396445bd32401d7c36eab6eb899db26f84087.